src/node/coinstats.h

```diff
@@ -23,6 +23,7 @@ enum class CoinStatsHashType {
 
 struct CCoinsStats
 {
+    CoinStatsHashType m_hash_type;
```

Comment:
9c8a265fd21a87228c18a1661df99fedc1866baf

`const CoinStatsHashType m_hash_type` and drop constructor.